common/app/layout/Sublink.scala

```diff
@@ -325,6 +326,12 @@ case class ContentCard(
     case _ => false
   }
 
+  def dataLinkName(index: Int): String = {
+    val name = s"$analyticsPrefix | card-@${index + 1}"
+    val withFromShowMore = if (fromShowMore) s"showmore | $name" else name
```

Comment:
I don't know anything about `data-link`. Is `showmore` supposed to be added as a prefix or appended at the end?